src/style.css

```
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  background-color: lightgrey;
+}
+
+.logo {
+  line-height: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 40px 0;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.navbar__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.navbar__item:last-child {
+  margin-right: 0;
+}
```

1. you can simplify with :not(:last-child)
2. no needs to set position: relative; to .navbar__item 
```suggestion
.navbar__item:not(:last-child) {
  margin-right: 20px;
}
```